src/node_native_module.cc

```diff
@@ -290,7 +183,7 @@ MaybeLocal<Function> NativeModuleLoader::LookupAndCompile(
     Local<Context> context,
     const char* id,
     std::vector<Local<String>>* parameters,
-    Environment* optional_env) {
+    NativeModuleLoader::Result* result) {
```

Comment:
This change probably indicates this needs to be split into two methods, `Lookup` and `Compile`, with no out-parameters.

Reply:
`Compile` would still have to return a `Maybe<Function>` and a `Result`, though, splitting it does not change that..

Reply:
I'll think about this some more...